app/scripts/directives/mobileClientBuildsRow.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+(function() {
+  angular.module('openshiftConsole').component('mobileClientBuildsRow', {
+    controller: [
+      'APIService',
+      'BuildsService',
+      'DataService',
+      'ListRowUtils',
+      'Navigate',
+      MobileClientBuildsRow
+    ],
+    controllerAs: 'row',
+    bindings: {
+      apiObject: '<',
+      buildConfig: '<'
+    },
+    templateUrl: 'views/mobile-client-builds-row.html'
+  });
+
+  function MobileClientBuildsRow(
+    APIService,
+    BuildsService,
+    DataService,
+    ListRowUtils,
+    Navigate
+  ) {
+    var row = this;
+    row.state = {};
+    row.buildConfigsInstantiateVersion = APIService.getPreferredVersion('buildconfigs/instantiate');
+    row.buildConfigsVersion = APIService.getPreferredVersion('buildconfigs');
+    var buildsVersion = APIService.getPreferredVersion('builds');
+    var watches = [];
+
+    _.extend(row, ListRowUtils.ui);
+
+    row.$onChanges = function(changes) {
+      var apiObjectChanges = changes.apiObject && changes.apiObject.currentValue;
```

Comment:
might be nicer to use the _.get here too, what do you think?

Reply:
:+1: